.gitignore

```
@@ -2,5 +2,6 @@ node_modules
 .next
 out
 /public/design
-/public/edit
+/public/edit.html
```

Actually, backward compatibility could be fixed but needs to be tested:

```suggestion
/public/edit*
```